pkg/operator/starter.go

```diff
@@ -224,6 +226,14 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		WithConditionalResources(bindata.Asset, []string{"assets/alerts/kube-apiserver-slos-extended.yaml"}, notOnSingleReplicaTopology, nil).
 		AddKubeInformers(kubeInformersForNamespaces)
 
+	performanceProfileInformerFactory := highcpuusagealertcontroller.PerformanceProfileDynamicInformer(dynamicClient)
```

Comment:
we already have an instance of `dynamicInformers` defined for this operator. Could we use it instead of creating a new one ?